index.js

```diff
@@ -20,7 +21,6 @@ const init = function (osseus) {
       false
     if (shouldUseMoleculerWebAsMiddleware) {
       try {
-        const OsseusMoleculerWeb = require('@colucom/osseus-moleculerweb')
```

Comment:
we still need to check the environment variable, it is possible we will want to run osseus-server without moleculer running as a middleware

Reply:
is that the reason why we require it only after the if?
we still don't initiate it if `!shouldUseMoleculerWebAsMiddleware` even after my change...